wordpress-mobile/WordPress-iOS
# Dashboard: Unified Menu Personalization

Part of #21492

- Implement Shortcuts (or Quick Actions) personalization. The settings are per-site.
- Add "Comments" quick action
- Reader shortcuts: "Stats" are not at the top

https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/812b72c4-7523-4f27-8d10-efd169a43e77

## To test:

- Open Personalization menu
- Change the settings and make sure the total number of enabled Quick Actions changes
- Return to Dashboard
- Verify that the Quick Actions are updated
- Verify that the height of the Quick Actions section got updated

### Card not Eligible

- Login with a blog that has no access to "Stats" (or update `func isEligible(for blog: Blog) -> Bool` to return `false`)
- Open Dashboard and verify that the "Stats" shortcut is not visible
- Open Personalization screen and verify that the "Stats" shortcut is not visible

## Regression Notes
1. Potential unintended areas of impact: Dashboard & Personalization
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
